lib/karafka/server.rb

```diff
@@ -3,6 +3,13 @@
 module Karafka
   # Karafka consuming server class
   class Server
+    @consumer_threads = Concurrent::Array.new
+
+    # How long should we sleep between checks on shutting down consumers
+    SUPERVISION_SLEEP = 1
```

Comment:
I feel that this warrants a configuration key

Reply:
@mach-kernel it just sleeps 1 second for each ```shutdown_timeout``` ```times``` to reach the shutdown_timeout timeout upon stale consumers. I don't see any benefit, especially as ```shutdown_timeout``` is configurable.